File: push/src/main/java/org/apache/thrift/protocol/TField.java
package org.apache.thrift.protocol;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:org/apache/thrift/protocol/TField.class */
public class TField {
    public final short id;
    public final String name;
    public final byte type;

    public TField() {
        this("", (byte) 0, (short) 0);
    }

    public TField(String str, byte b, short s) {
        this.name = str;
        this.type = b;
        this.id = s;
    }

    public TField(String str, byte b, int i) {
        this(str, b, (short) i);
    }

    public boolean equals(TField tField) {
        return this.type == tField.type && this.id == tField.id;
    }

    public String toString() {
        return "<TField name:'" + this.name + "' type:" + ((int) this.type) + " field-id:" + ((int) this.id) + ">";
    }
}

File: push-protocol-frozen/src/main/java/org/apache/thrift/protocol/TField.kt
package org.apache.thrift.protocol

class TField @JvmOverloads constructor(
    @JvmField val name: String = "",
    @JvmField val type: Byte = 0,
    @JvmField val id: Short = 0,
) {
    constructor(name: String, type: Byte, id: Int) : this(name, type, id.toShort())

    fun equals(tField: TField): Boolean {
        return type == tField.type && id == tField.id
    }

    override fun toString(): String {
        return "<TField name:'$name' type:${type.toInt()} field-id:${id.toInt()}>"
    }
}
